Panic on nil map when labels exist only in second config

Version: `1.22.0 (HEAD)`

`$ kompose -f 1.yml -f 2.yml convert`

1.yml:
```yml
version: '3.7'

services:
    a:
      image: "alpine:latest"
```

2.yml:
```yml
version: '3.7'

services:
    a:
      labels:
          test: "value"
```

Looking at the code, `mergeComposeObject` assumes that the 'old' service will always have labels. See [v3.go#L689](https://github.com/kubernetes/kompose/blob/76565d80b2dccfa2de4b4612557788b1861cc48a/pkg/loader/compose/v3.go#L689)

The quick fix here is to give `a` the `labels: []` key in 1.yml.

```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/kubernetes/kompose/pkg/loader/compose.mergeComposeObject(0xc0000329c0, 0xc000033f80, 0xc0002feea0, 0x27, 0xc0004d7f60)
	github.com/kubernetes/kompose/pkg/loader/compose/v3.go:691 +0x1ba9
github.com/kubernetes/kompose/pkg/loader/compose.parseV3(0xc000438300, 0x2, 0x2, 0x3, 0x1, 0x1, 0x1, 0x700, 0x680)
	github.com/kubernetes/kompose/pkg/loader/compose/v3.go:116 +0x445
github.com/kubernetes/kompose/pkg/loader/compose.(*Compose).LoadFile(0x559f14e88110, 0xc000438300, 0x2, 0x2, 0x0, 0x0, 0x7f60bc057080, 0x8, 0x10, 0x559f138f282a)
	github.com/kubernetes/kompose/pkg/loader/compose/compose.go:180 +0x29a
github.com/kubernetes/kompose/pkg/app.Convert(0x100000100, 0x0, 0x0, 0x0, 0x0, 0x559f1441bdb9, 0x4, 0x1, 0x559f1442ca8d, 0x15, ...)
	github.com/kubernetes/kompose/pkg/app/app.go:224 +0x126
github.com/kubernetes/kompose/cmd.glob..func3(0x559f14df2e20, 0xc00034d780, 0x0, 0x4)
	github.com/kubernetes/kompose/cmd/convert.go:106 +0x58
github.com/spf13/cobra.(*Command).execute(0x559f14df2e20, 0xc00034d740, 0x4, 0x4, 0x559f14df2e20, 0xc00034d740)
	github.com/spf13/cobra@v1.0.0/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x559f14df30c0, 0x559f1388206c, 0x559f14e0a780, 0xc000000180)
	github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.0.0/command.go:887
github.com/kubernetes/kompose/cmd.Execute()
	github.com/kubernetes/kompose/cmd/root.go:92 +0x33
main.main()
	github.com/kubernetes/kompose/main.go:22 +0x25
```

--- a/pkg/loader/compose/v3.go
+++ b/pkg/loader/compose/v3.go
@@ -747,6 +747,9 @@ func mergeComposeObject(oldCompose *types.Config, newCompose *types.Config) (*ty
 		}
 		if len(service.Labels) != 0 {
 			// merge the 2 sets of values
+			if tmpOldService.Labels == nil {
+				tmpOldService.Labels = make(map[string]string)
+			}
 			for k, v := range service.Labels {
 				tmpOldService.Labels[k] = v
 			}

